Dockerfiles/prod/data/docker-entrypoint.d/311-project-startup-scripts.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+############################################################
+# Functions
+############################################################
+
+###
+### Execute project-supplied scripts
+###
+execute_project_scripts() {
+  local debug="${1}"
+  local script_dir
+  local script_name
+  local DEVILBOX_PROJECT_DIR
+  local DEVILBOX_PROJECT_NAME
+  local DEVILBOX_PROJECT_DOCROOT
+
+  for DEVILBOX_PROJECT_DIR in /shared/httpd/*; do
+
+    script_dir=${DEVILBOX_PROJECT_DIR}/.devilbox/autostart
```

Comment:
The PHP-FPM images are not aware about the `.devilbox/` directory. This is specific to the HTTPD images and can also be changed to any other directory:

https://github.com/devilbox/docker-nginx-mainline/blob/master/doc/environment-variables.md#-mass_vhost_template_dir

So hardcoding it here would not work.

In other words, the PHP-FPM images don't know about this path and hardcoding it here would result in it not working, once you change it via the HTTPD images (or in our use-case vie the Devilbox `.env` variable)

Reply:
I see.

Would it be an option to propagate the `MASS_VHOST_TEMPLATE_DIR` to the PHP-FPM images too and use it in the script? Or create a new variable that defaults to the same directory?

Reply:
> Would it be an option to propagate the `MASS_VHOST_TEMPLATE_DIR`

This would not make much sense, as the PHP images have nothing to do with it.

Reply:
@cytopia Actually, You are already populating the `MASS_VHOST_TEMPLATE_DIR` to the `php` image in the form of `HTTPD_TEMPLATE_DIR` and you're even using it yourself in [your code](https://github.com/cytopia/devilbox/blob/986f04637f8210e18e2895d01888040e384c662a/.devilbox/www/htdocs/index.php#L472) for the status page.

I have updated the code in order to use the value of that variable.